msbrainpy/atlas/query.py

```diff
@@ -142,3 +146,165 @@ def generateQueryRows(model, criteria, options='[only$eq''genes.entrez_id,data_s
         print('new start row: {}'.format(startRow))
         done = retrieved == 0
         yield queryResult
+
+
+
+# ----------------------------------------------- Get Gene Images ------------------------------------------------------
+def download_gene_images(entrez_id, directory_path=None, age_id=15, product_id=1, verbose=False):

```

Comment:
You should also follow PEP8 recommendations, ie line length max 80c. You can set up your editor to remind you of these things. What editor do you use?

Reply:
You should also give all your functions a docstring using the NumPy formatting standard:

https://numpy.org/doc/stable/docs/howto_document.html

That will make it clear to readers such as myself what `product_id` refers to, and what its possible values are.